lib/codecs/src/encoding/format/json.rs

```diff
@@ -33,12 +45,15 @@ impl JsonSerializerConfig {
 
 /// Serializer that converts an `Event` to bytes using the JSON format.
 #[derive(Debug, Clone)]
-pub struct JsonSerializer;
+pub struct JsonSerializer {
+    #[allow(dead_code)]
```

Comment:
🤔  maybe fog from time off but, seems like this should be removed?

Reply:
Fixed in 49e470b